src/script/conversation/ConversationState.ts

```diff
@@ -195,6 +195,14 @@ export class ConversationState {
         });
   }
 
+  /**
+   * indicate whether the selfUser has a conversation with this other user
```

Comment:
```suggestion
   * indicate whether the selfUser has a one one one conversation with this other user
```
Question: is this correct?
Just making sure if a clarification is needed to differentiate user known through group conversation but not connected

Reply:
no, it's actually any kind of conversation (including group conversations). We can have unconnected users there (as opposed to what the `connectedUsers` suggests 😬 )

Reply:
```suggestion
   * indicate whether the selfUser has a one on one conversation with this other user
```
Question: is this correct?
Just making sure if a clarification is needed to differentiate user known through group conversation but not connected